CPackLists.txt

```diff
@@ -64,27 +67,33 @@ IF(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
    message( STATUS "make package" )
 ENDIF()
 
-
-message( STATUS "\nOption to install using 'sudo make install" )
+IF(NOT MINGW)
+   message( STATUS "\nOption to install using 'sudo make install'" )
+ELSE()
+   message( STATUS "\nOption to install using 'make install'" )
+ENDIF()
 message( STATUS "Installation locations: " )
 message( STATUS "====================" )
 message( STATUS "Headers: ${CPACK_INSTALL_PREFIX}/${CMAKE_INSTALL_HEADERDIR}" )
 message( STATUS "Library installation directory: ${CPACK_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}" )
 
 message( STATUS "For more information please see g3log/CPackLists.txt\n\n" )
-message( STATUS "To install: sudo dpkg -i g3log-***Linux.deb" )
-message( STATUS "To list package contents: sudo dpkg --contents g3log-***Linux.deb" )
-
-message( STATUS "List content of the installed package: sudo dpkg -L g3log" )
-message( STATUS "To remove: sudo dpkg -r g3log" )
+IF(NOT MINGW)
+   message( STATUS "To install: sudo dpkg -i g3log-***Linux.deb" )
+   message( STATUS "To list package contents: sudo dpkg --contents g3log-***Linux.deb" )
 
+   message( STATUS "List content of the installed package: sudo dpkg -L g3log" )
+   message( STATUS "To remove: sudo dpkg -r g3log" )
+ENDIF()
 #  NOTE: to change installation locations you can use the settings below
 #  examples:
 #  CPACK_INSTALL_PREFIX
 #  CPACK_PACKAGE_INSTALL_DIRECTORY
+#  CMAKE_INSTALL_PREFIX
 #  CMAKE_INSTALL_HEADERDIR
 #  CMAKE_INSTALL_LIBDIR
 
-
-INCLUDE(CPack)
+IF(NOT MINGW)
+   INCLUDE(CPack)
```

Comment:
Does CPack not exist on `mingw`? When searching for `mingw cpack CPackLists.txt`. I get several hits from CMake and StackOverflow so I think you should be able to use CPack in your environment

Reply:
I thought this module is needed for deb package generation.
When I include it under MinGW there is target 'package' in the Make file, but it gives me error about missing NSIS. Now I have installed it and now there is new the error:

CMake Error at D:/g3log/build/cmake_install.cmake:54 (message):
  ABSOLUTE path INSTALL DESTINATION forbidden (by caller):
  D:/g3log/install/include/g3log/active.hpp;D:/g3log/install/include/g3log/atomicbool.hpp;D:/g3log/install/include/g3log/crashhandler.hpp;D:/g3log/install/include/g3log/filesink.hpp;D:/g3log/install/include/g3log/future.hpp;D:/g3log/install/include/g3log/g3log.hpp;D:/g3log/install/include/g3log/logcapture.hpp;D:/g3log/install/include/g3log/loglevels.hpp;D:/g3log/install/include/g3log/logmessage.hpp;D:/g3log/install/include/g3log/logworker.hpp;D:/g3log/install/include/g3log/moveoncopy.hpp;D:/g3log/install/include/g3log/shared_queue.hpp;D:/g3log/install/include/g3log/sink.hpp;D:/g3log/install/include/g3log/sinkhandle.hpp;D:/g3log/install/include/g3log/sinkwrapper.hpp;D:/g3log/install/include/g3log/stacktrace_windows.hpp;D:/g3log/install/include/g3log/std2_make_unique.hpp;D:/g3log/install/include/g3log/stlpatch_future.hpp;D:/g3log/install/include/g3log/time.hpp;D:/g3log/install/include/g3log/g2log.hpp;D:/g3log/install/include/g3log/generated_definitions.hpp